src/dos/dos_code_page.cpp

```diff
@@ -2046,6 +2050,91 @@ static std::optional<uint16_t> find_ega_font_storage_index(const uint16_t code_p
 	return {};
 }
 
+static void patch_font_dotted_i(ScreenFont& font)
+{
+	// Incomplete fonts should not be allowed in storage
+	assert(font.font_8x16.size() == ScreenFont::FullSize_8x16);
+	assert(font.font_8x14.size() == ScreenFont::FullSize_8x14);
+	assert(font.font_8x8.size()  == ScreenFont::FullSize_8x8);
+
+	// Codes of characters to swap
+	constexpr uint8_t CodePoint1 = 0x49;
+	constexpr uint8_t CodePoint2 = 0xf2;
```

Comment:
Maybe explain in a comment which characters are these.

Reply:
OK, added comments.